px-vis-interaction-space.html

```diff
@@ -763,6 +766,38 @@
 
     _searchOnWebWorker: function(mousePos, dataObj) {
       var ctx = {};
+      var registerDataObj = {};
+      var registerCtx = {};
+
+      this._calcTooltipContextData(registerCtx, mousePos);
+
+      this._calcTooltipContextData(ctx, mousePos);     
+    
+      registerCtx.successCallback = function(e) {
+        if(!e.data) { return; }
+
+        this._populateTooltipDataObject(registerDataObj, e);       
+        
+      }.bind(this);
+
+      ctx.data.searchFor = this.searchFor;
+
+      ctx.successCallback = function(e) {
+        if(!e.data) { return; }
+
+        this._populateTooltipDataObject(dataObj, e);
+        dataObj.registerTooltipData = registerDataObj;		    
+
+        this._calcDataFinished(dataObj);
+      }.bind(this);
+
+      Px.vis.scheduler.process(registerCtx);
+      Px.vis.scheduler.process(ctx);
+    },
```

Comment:
Could you please write more comments, why do we need this fix? 
Try adding the defect number in the PR and comment

Reply:
As discussed, please add a flag to enable or disable this functionality by boolean flag, something like `showTooltipPointsMode`